packages/core/src/html/MdAttributeRenderer.js

```diff
@@ -35,6 +35,12 @@ class MdAttributeRenderer {
         rendered = this.markdownProcessor.renderMd(node.attribs[attribute]);
       }
 
+      if (slotName === 'header' && _.has(node.attribs, 'minimized')) {
```

Comment:
> This needs some discussion. Simply changing `<p>` to `<span>` won't work (many other block tags), it might also break the formatting / browser tag placement drastically.

Yup, thanks for bringing this to my attention. I was rather naive in that implementation and didn't consider these possibilities. 

> The general issue here is the author is free to write block html in both header (a must) and alt - i.e. this is again a similar definition issue to #1548.

As discussed offline, there are two possible sources of hydration issues when we are dealing with **minimized** panels (regardless of whether it is `MinimalPanel` or `NestedPanel`). These two sources of hydration issues come from `header` and `_alt` slots. 

It seems like `_alt` slot for now won't pose a hydration issue due to an edge case of SSR. Typically when the user writes block html in `_alt` slot when using minimized panel, it should cause a hydration issue. But there is an edge case in SSR such that when `_alt` slot is wrapped within `<button>` tag as shown below (as per our current code), this hydration issue can be avoided. 

```
      <button class="morph-display-wrapper btn card-title morph-title" @click="minimalOpen()">
        <slot name="_alt">
          <slot name="header"></slot>
        </slot>
      </button>
```

I believe this is because `<button>` tag belongs to a "grey" area as it is considered an *inline-block*. Thus, when using minimized panel, the user can write block html in either `_alt` or `header` slot and it won't trigger hydration issue as long as those tags are wrapped in `<button>` tag. 

This also means that there should be no problem as of now. Nevertheless, it is still good to define the behavior as you brought up. I think both of the solutions you suggested are good. As per your suggested solution: (1) "enforce inline rendering for alt", (2) "tighten #1548's definition of an inline panel to <panel minimized alt="...">".

I'm thinking that (2) should be done in another PR, but should we do (1) within this PR? Or should we do (1) in another PR as well?